cmd/pretriage/main.go

```diff
@@ -13,7 +13,16 @@ import (
 	"github.com/shiftstack/bugwatcher/pkg/query"
 )
 
-const queryUntriaged = query.ShiftStack + `AND assignee is EMPTY AND (labels not in ("Triaged") OR labels is EMPTY)`
+const queryUntriaged = query.ShiftStack + `
+	AND (
+		(
+			assignee is EMPTY AND (labels not in ("Triaged") OR labels is EMPTY)
+		)
+		OR (
+			assignee = "shiftstack-dev@redhat.com"
+		)
+	)
```

Comment:
Because of these parens, the query will still check for the component before assigning. Bugs with a component that is not ours will be ignored and could be forgotten. I don’t know why a bug with the wrong component would be assigned to our service account, but if that happens I’d like us to notice.

I would remove this outer parens, so that “OR assigned-to-the-bot” applies to ANY bug assigned to the bot.

Reply:
Incidentally, that's not the actual Jira name of the bot